execution_workflows/GETUTR/modules/postprocessing_relative_quantification.nf

```diff
@@ -0,0 +1,26 @@
+// Import generic module functions
+include { initOptions; saveFiles; getSoftwareName } from './functions'
+
+params.options = [:]
+def modules = params.modules.clone()
+// get the configs for this process
+
+/*
+    Convert GETUTR output file to relative quantification challenge file
+*/
+process POSTPROCESSING_RELATIVE_QUANTIFICATION {
+    publishDir "${params.outdir}/getutr", mode: params.publish_dir_mode
+    container "docker.io/apaeval/getutr:latest"
+
+    input:
+    tuple val(sample), path(getutr_output)
+
+    output:
+    path "*"
+
+    script:
+    relative_quantification_output = "${sample}_relative_quantification_output.bed"
```

Comment:
Can you make the file suffix a parameter in the config? By default we will likely save it as sample_04.bed.

Reply:
sure!